runtime/common/src/parachains.rs

```diff
@@ -59,11 +59,30 @@ use inherents::{ProvideInherent, InherentData, MakeFatalError, InherentIdentifie
 
 use system::{
 	ensure_none, ensure_signed,
-	offchain::SubmitSignedTransaction,
+	offchain::{CreateSignedTransaction, SendSignedTransaction, Signer},
 };
 use crate::attestations::{self, IncludedBlocks};
 use crate::registrar::Registrar;
 
+pub const KEY_TYPE: KeyTypeId = KeyTypeId(*b"para");
```

Comment:
@rakanalh  Please follow up - there are missing docs here and below. Also, this is probably duplicated from `primitives/src/parachain`. Why not define this for `ValidatorId` or the `fish` key?

Reply:
I originally wanted to do this but somehow forgot to. Will follow up in a new PR.